app/assets/javascripts/beak/widgets/accessibility/key-combo.coffee

```diff
@@ -0,0 +1,40 @@
+import { isMac } from "./utils.js"
+
+export class KeyCombo
+  # (String) => Unit
+  constructor: (comboStr) ->
+    @comboStr = comboStr
+    parts   = comboStr.toLowerCase().split('+').map((x) => x.trim())
+    @key    = parts.pop()
+    @modifiers = {
+      ctrl:  parts.includes('ctrl')  or parts.includes('control')
+      , alt:   parts.includes('alt')   or parts.includes('option')
+      , shift: parts.includes('shift')
+      , meta:  parts.includes('meta')  or parts.includes('cmd') or parts.includes('command') or parts.includes('mod')
+    }
+
+  # Print methods
+  # { String: String }
+  keyStringTable: {
+    'up': "↑",
+    'down': "↓",
+    'left': "←",
+    'right': "→",
+    'escape': "Esc",
+  }
+
+  # Array[String]
+  getKeys: -> [
+    ...Object.entries(@modifiers).filter(([_, v]) => v).map(([mod, _]) =>
```

Comment:
I consider ```coffee
  ...some.long.multiline(
    computation
  ),
  if pred then x else y
]
```

to really lack in clarity and readability.  I'd really prefer you to store these complex values into variables, and then assemble the output array from the variables.  I don't usually value concision or cleverness in code.